src/js/main.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-env browser */
+/* eslint-env browser */
```

Comment:
I don't think we need this line twice 🙂

Reply:
I have deleted the duplicate line. Thank you for noting.